packages/integrations/sitemap/test/custom-sitemaps.test.js

```diff
@@ -0,0 +1,29 @@
+import assert from 'node:assert/strict';
+import { before, describe, it } from 'node:test';
+import { sitemap } from './fixtures/static/deps.mjs';
+import { loadFixture, readXML } from './test-utils.js';
+
+describe('Custom sitemaps', () => {
+  /** @type {import('./test-utils.js').Fixture} */
+  let fixture;
+  /** @type {string[]} */
+  let urls;
+
+  before(async () => {
+    fixture = await loadFixture({
+      root: './fixtures/static/',
+      integrations: [
+        sitemap({
+          customSitemaps: ['http://example.com/custom-sitemap.xml'],
```

Comment:
Can you please add more use cases? The zod schema says that these are URLs, this means that we can have `file://some/path.js` or `https://foo.com`. I don't know if they are correct uses cases or not. Probably not. If they aren't, we should add another level of validation. If they are, we should add them to the test cases

Reply:
I used the same schema as [`customPages`](https://github.com/withastro/astro/blob/3fcb869ed61b5cf2f994756dafb62feff3fbeb94/packages/integrations/sitemap/src/schema.ts#L10-L11), should that also be updated?

Reply:
No, you're focusing on your features, and we're discussing the code you're writing  :) 

You could send another PR to update custom pages if you want